packages/kotti-ui/source/kotti-filter/components/FilterButtonLink.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+	<div class="kt-filter-button-link">
+		<span @click="handleClick">
+			<i class="yoco" v-text="icon" /> {{ label }}
+		</span>
+	</div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api'
+
+export default defineComponent<{ label: string; icon: string }>({
+	name: 'FilterButtonLink',
+	props: {
+		label: {
+			type: String,
+			default: '',
+		},
+		icon: {
+			type: String,
+			default: '',
+		},
+	},
+	setup(props, { emit }) {
+		return {
+			handleClick(event) {
+				emit('click', event)
+			},
+		}
+	},
+})
+</script>
+
+<style lang="scss" scoped>
+@import '../../kotti-style/_variables.scss';
+
+.kt-filter-button-link {
```

Comment:
I think this should be `.kt-filter__button-link` in BEM, but not 100% sure as it’s in a separate component